modules/contributor-guide/partials/ref_che-theia-plug-in-api.adoc

```diff
@@ -51,6 +51,27 @@ This namespace contains APIs and types related to {prod-short}. Some sections ar
 | workspace | Allows to work with the {prod-short} workspace API; provides functionality to access workspaces, their lifecycle, and user settings.
 |===
 
+[id="using-the-{prod-id-short}--in-plug-ins_{context}"]
+== Using the `che` {orch-namespace} in plug-ins
+
+VS Code extensions::
++
+. Adding the `@eclipse-che/plugin` dependency for backend plug-ins to `package.json` file is a possibility.
+. But for quick calls, it's simpler to use VS Code extension mechanism because it does not require to rely on 3rd party dependency or try and catching the error.
```

Comment:
```suggestion
. For quick calls, it is simpler to use VS Code extension mechanism because it does not require relying on a third-party party dependency or trying and catching the error.
```

Reply:
thanks, applied